### ISSUE TITLE
Option to link-back to mastodon toot

### ISSUE THREAD

**Issue by taoeffect** [ineffyble/tooter#8]
A link to the original mastodon toot in the tweet, so as to help migrate people off of twitter, because it engages in rampant and hard-to-detect censorship & throttling. Ideally this option would be enabled by default.

**Comment by ineffyble** [comment]
Hmm. So when "Toot and Tweet" is clicked, you want the text on the Twitter status to include a link to the Toot?

**Comment by taoeffect** [comment]
Yeah, and I've reconsidered it being enabled-by-default, but it would still be a useful feature. If the text is too long, it can automatically calculate that for the tweet and add ellipses to the tweet-version of it, followed by the link. :)
